spelling_wordlist.txt

```diff
@@ -112,6 +116,7 @@ satisfiability
 satisfiable
 scalable
 se
+sharable
```

Comment:
shareable is a real word I think, unlike sharable

Reply:
absolutely right, removed